PendingReleaseNotes.md

```diff
@@ -2,5 +2,13 @@
 
 ## Breaking Changes
 
+* The mds probes is now been handled by filesystem CR instead of cluster CR, now to apply configuration for the mds probes we need specify in filesystem CR,
```

Comment:
I
```suggestion
* The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](ceph-filesystem-crd.md) for more details. 
```